README.markdown

```diff
@@ -231,6 +250,14 @@ Check all signed certificates (including the CA certificate) for certificates th
   Expires in: 4 minutes, 19 seconds
 ~~~
 
+#### puppet certregen cacert
+
+Regenerate the CA certificate with the subject of the old CA certificate and updated notBefore and notAfter dates.
+
+#### puppet certregen crl
+
+Update the CRL with a nextUpdate field of 5 years from the present date.
+
```

Comment:
This is where I'd add your explanation: 

> CRLs don't usually expire, since their expiration date is normally updated whenever a certificate is revoked. But in rare cases, like when no certificates have been revoked in five years, they can expire and cause problems.